core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -498,7 +498,8 @@ private void abortBlockInternal(long sessionId, long blockId) {
    * @param pinOnCreate is block pinned on create
    * @return destination location to move the block
    */
-  private BlockStoreLocation commitBlockInternal(long sessionId, long blockId,
+  @VisibleForTesting
+  public BlockStoreLocation commitBlockInternal(long sessionId, long blockId,
```

Comment:
try seeing if it's possible to remove the public qualifier

Reply:
Removed